src/components/Charts/ChataChart/ChataChart.js

```diff
@@ -192,9 +191,35 @@ export default class ChataChart extends Component {
     return { chartHeight, chartWidth }
   }
 
+  sortChartDataByDate = (props) => {
+    try {
+      if (!props.data || typeof props.data !== 'object') {
```

Comment:
Is there a specific reason for these 2 checks and not just something like `props.data?.length`?